### DISCUSSION TITLE
Multipar 3

### DISCUSSION THREAD

**Discussion by Gitt-hubb** [Yutaka-Sawada/MultiPar#122] [Q&A] [answered]
Any plans for releasing Mutipar 3 for windows,  and secondary will there be 64bit versions available or for current beta version. 😀

**Comment by Yutaka-Sawada** [comment, marked as answer]
At this time, I'm waiting refine of PAR3 specification. I found a problem in current design of PAR3, which may not detect integrity of PAR3 files themselves. This problem exists in PAR2, too. PAR3 doesn't solve it yet.

For example, you makes two PAR2 files. When you join them into one PAR2 file, the resulting PAR2 file becomes complete still. In this case, PAR2 cannot detect the difference of original PAR2 files and combined PAR2 file.
